string void_request
---
uint32 window_colordepth
